### DISCUSSION TITLE
Config.toml

### DISCUSSION THREAD

**Discussion by blogdemoi** [hrkfdn/ncspot#1169] [Q&A]
Hu,

I just started using Ncspot, however I have problems with the config.toml.

I can change the theme, but can't change the initial_screen parameter. Whatever I use, e.g. `initial_screen = "search",` it doesn't work, I always start with the library tab.

Thank you.

**Comment by Bettehem** [comment]
Hi, if you think this is a bug in ncspot, it would be better to report it on the issues page here: https://github.com/hrkfdn/ncspot/issues
